quiz.js

```diff
@@ -0,0 +1,76 @@
+checkHeight();
+var catchem = document.getElementById('catchem');
+catchem.addEventListener("click", handleClicks);
+catchem.addEventListener("keyup", handleKeyUp);
+
+function handleKeyUp (event) {
+	if (event.keyCode == 13) {
+		checkHeight();
+		
+	}
+}
+
+
```

Comment:
Be careful about consistent spacing between blocks of code. One line should suffice